pkgs/tools/networking/strongswan/default.nix

```diff
@@ -29,9 +30,10 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ pkgconfig autoreconfHook ];
   buildInputs =
-    [ curl gmp python iptables ldns unbound openssl pcsclite ]
+    [ curl gmp python ldns unbound openssl pcsclite ]
     ++ optionals enableTNC [ trousers sqlite libxml2 ]
-    ++ optionals stdenv.isLinux [ systemd.dev pam ]
+    ++ optionals stdenv.isLinux [ systemd.dev pam iptables ]
+    ++ optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [ CoreFoundation SystemConfiguration ])
```

Comment:
CoreFoundation should be provided by stdenv so you should just need SystemConfiguration here.

Reply:
Thanks, fixed. Was not aware that stdenv provides CoreFoundation.